ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala

```diff
@@ -113,11 +134,13 @@ object DefaultComponentService {
   }
 }
 
-class DefaultComponentService private (config: Config,
-                              processingTypeDataProvider: ProcessingTypeDataProvider[ProcessingTypeData],
-                              fetchingProcessRepository: FetchingProcessRepository[Future],
-                              subprocessRepository: SubprocessRepository,
-                              categoryService: ConfigProcessCategoryService)(implicit ec: ExecutionContext) extends ComponentService {
+class DefaultComponentService private(componentsIdStorage: Map[ComponentId, String],
```

Comment:
what is this mapping? Can we name it like componentIdsTo??

Reply:
I will add  some information about it in comment